include/fmt/compile.h

```diff
@@ -37,6 +37,28 @@ struct is_compiled_string : std::is_base_of<compiled_string, S> {};
  */
 #define FMT_COMPILE(s) FMT_STRING_IMPL(s, fmt::detail::compiled_string)
 
+#if defined(__cpp_nontype_template_parameter_class) && \
+    (!FMT_GCC_VERSION || FMT_GCC_VERSION >= 903)
+template <typename Char, std::size_t N> struct fixed_string {
```

Comment:
std::size_t -> size_t for consistency with the rest of the library.

Reply:
Done